### DISCUSSION TITLE
Revert your Chromebook to 110 to install crouton

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#4968] [General]
Normally, ChromeOS updates are welcomed and serendipitous. However, in the case of crouton, updates frequently caused crouton to break. At which point, the crouton devs would step in, update crouton, then you could fix your chroot by updating it via crouton. Until recently, that is...

As of version 111, autotools is no longer available. This 3rd party component was used by crouton to compile _cras_, the ChRome Audio Server. Without autotools, your cannot install a crouton desktop.

One work around has been provided by @dnschneid, namely, a crouton branch called "silence". I recently covered this in a previous article titled,  [The future of crouton: an errie silence](https://github.com/dnschneid/crouton/discussions/4967) .

Another work around is to temporarily revert your Chromebook to version 110. Then, you will be able to install your crouton desktop, including the compiling of cras. After you've been online for a few hours, ChromeOS will offer to update your computer to 111. Even after the update, your chroot will continue to work.

If  you're on the beta or dev channel, you first need to return to stable. Then, assuming your at version 111, follow these steps to revert to 110:

**Reboot your Chromebook. When you're at the login screen, press Ctrl-Shift-Alt-R. Wait until you see the Powerwash screen, then press Ctrl-Shift-Alt-R again. Confirm that you wish to Revert and Powerwash. Local data will be cleared. When you're able to login again, you'll be able to install crouton normally.**

If the above doesn't work, e.g., you've already updated to 112, then you can install an earlier version of ChromeOS by using the Chromebook Recovery Utility. Download version 110 from [ChromeOS Versions](http://cros.tech).

**Comment by satmandu** [comment]
I wonder if users could just use the autotools from Chomebrew to install cras...

**Reply by CroutonIsFun** [reply]
@satmandu

I think this file has to be on the server, not stored locally. However, the script could pull it from Chromebrew, maybe. Really interesting suggestion.

**Reply by satmandu** [reply]
Happy to help ...

We also have a `cras` package...

**Reply by CroutonIsFun** [reply]
Really? Awesome!
